src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/services/AuthService.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.services;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.authorisation.validators.AuthTokenValidator;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.services.exception.ForbiddenException;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.services.exception.UnauthenticatedException;
+
+import static java.util.Arrays.asList;
+import static org.apache.commons.lang.StringUtils.isBlank;
+
+@Component
+public class AuthService {
+
+    private final AuthTokenValidator authTokenValidator;
+    private final String[] allowedServices;
```

Comment:
would a `List` work?

Reply:
I had to change the YAML property value to comma-separated values. The YAML list property is not working with `@Value` annotation. 
A property having `comma-separated values` can be mapped with `String[]` so no need to use utility methods to convert from `String` to `Collection/Array`.